check-for-missing-vals.py

```
@@ -0,0 +1,7 @@
+# Check for missing values
+print(data.isnull().sum())
```

Ruff (0.12.2)

Undefined name `data`

(F821)